library/std/src/io/error.rs

```diff
@@ -156,6 +156,11 @@ pub enum ErrorKind {
     /// [`Ok(0)`]: Ok
     #[stable(feature = "rust1", since = "1.0.0")]
     WriteZero,
+    /// The underlying storage (typically, a filesystem) is full.
+    ///
+    /// This does not include out of quota errors.
+    #[stable(feature = "io_error_no_storage_space", since = "1.51.0")]
```

Comment:
1.47 is stable and 1.49 is the current nightly.
```suggestion
    #[stable(feature = "io_error_no_storage_space", since = "1.49.0")]
```

Reply:
Right.  I'm a bit pessimistic about when this might go in but if it seems right in principle I will change it obviously.